tensorflow/inference/docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -36,11 +42,16 @@
 SAGEMAKER_TFS_PORT_RANGE = os.environ.get("SAGEMAKER_SAFE_PORT_RANGE")
 TFS_INSTANCE_COUNT = int(os.environ.get("SAGEMAKER_TFS_INSTANCE_COUNT", "1"))
 
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(format='%(process)d %(asctime)s %(levelname)-8s %(message)s', force=True, level=logging.INFO)
 log = logging.getLogger(__name__)
 
 CUSTOM_ATTRIBUTES_HEADER = "X-Amzn-SageMaker-Custom-Attributes"
 
+# Create a shared dictionary using a manager
+manager = Manager()
+shared_tfs_instance_dict = manager.dict()
+lock = manager.Lock()
+
```

Comment:
Don't we need to avoid creating a `multiprocessing.Manager()` if we're using `gevent`?

Probably good to add a quick validation here in this file, too, that ensures we don't somehow end up with `gevent` in MME mode.

Reply:
As long as we don't call it. It will be fine.